src/views/project/publishPartition/SummaryView.vue

```diff
@@ -0,0 +1,162 @@
+const projectId = route.params.id
+const partitionId = route.params.partitionId
+const isLoaded = ref(false)
+
+let partition = null
+let taxa = ref(0)
+let characters = ref(0)
+let onetimeMedia = ref([])
+let media = ref(null)
+let labels = ref(0)
+let documents = ref(0)
+let bibliographicReferences = ref(0)
+let views = ref(0)
+let specimens = ref(0)
+
+onMounted(async () => {
+  const url = `${
+    import.meta.env.VITE_API_URL
+  }/projects/${projectId}/publish/partition/${partitionId}`
+
+  const response = await axios.get(url)
+
+  partition = response.data.partition
+  taxa.value = response.data.taxaLength
+  characters.value = response.data.characterLength
+  onetimeMedia.value = response.data.onetimeMedia
+  media.value = response.data.mediaLength
+  labels.value = response.data.labels
+  documents.value = response.data.documents
+  bibliographicReferences.value = response.data.bibliographicReferences
+  views.value = response.data.views
+  specimens.value = response.data.specimens
+
+  isLoaded.value = true
+})
+
+async function publishPartition(event) {
+  const formData = new FormData(event.target)
+  const formObject = Object.fromEntries(formData.entries())
+
+  const onetimeAction = formObject.onetimeAction || 1
+
+  const url = `${
+    import.meta.env.VITE_API_URL
+  }/projects/${projectId}/publish/partition/${partitionId}`
+  const response = await axios.post(url, { onetimeAction })
+
+  if (response.status == 200) {
+    alert('Publishing Partition')
+    router.push({ path: `/myprojects/${projectId}/overview` })
+  } else {
+    alert('Could not publish Partition')
+  }
+}
+</script>
+<template>
+  <LoadingIndicator :isLoaded="isLoaded">
+    <RouterLink :to="`/myprojects/${projectId}/publish/partition`">
+      <button type="button">Go Back</button>
+    </RouterLink>
+
+    <h1>Partition: {{ partition.name }}</h1>
+    <p>
+      This partition contains {{ characters }} characters, {{ taxa }} taxa,
+      {{ media }} media of which {{ onetimeMedia.length }} are released for
+      onetime use, {{ labels }} media labels, {{ specimens }} specimens,
+      {{ views }} views, {{ documents }} documents that are linked to media
+      explaining copyright and {{ bibliographicReferences }} bibliographic
+      references To edit these, go to the Matrix Editor and make changes
+    </p>
+    <p>
+      When you publish this partition, a new project will be created with the
+      characters and taxa listed above. Any matrices, media, copyright documents
+      and bibliographic citations associated with the characters and taxa will
+      be copied into the new project. The new project will be left in an
+      unpublished state to allow you time to review prior to publication.
+    </p>
+    <p>
+      Note that the new project you create here will be separate from P4091
+      (Test) and create only the subset of information described above. Changes
+      made to the newly created project will not affect P4091 or vice versa.
```

Comment:
This paragraph referenced P4091. This should be replaced with the id and name of the current project.